content/en/cloud_cost_management/google_cloud.md

```diff
@@ -0,0 +1,124 @@
+
+<div class="alert alert-warning"> LEGACY project integrations are deprecated and not supported </div>
+
+Datadog Google Cloud Platform Integrations monitor the entire project when a related service account is integrated. Using a previously integrated project prevents monitoring resources in a new project. If your billing is associated with a non-integrated project, those resources will start to be monitored.
+
+**Note:** You can limit Metrics Collection per integration - this limits automatically monitoring resources in the project - but does not exclude data from Cloud Cost processing. </div>
+
+### Provide Datadog integration service account necessary permissions
+The APIs and permissions below enable Datadog to access your Detailed Usage billing export data and extract it in a useful format. This dumps data from BigQuery, where Google Cloud exports it, to your specified GCS bucket through a scheduled BigQuery query. The data is exported in a useful manner in your cloud storage bucket where it can then be processed by Datadog.
+
+#### Enable necessary Google Service APIs
+[Enable BigQuery and BigQuery Data Transfer Service APIs][5] if not already enabled
+
+#### Configure required bucket access
+[Add the service account as a principal on the GCS bucket resource][6] and assign a role with the following permissions:
+* `storage.buckets.get`
+* `storage.objects.create`
+* `storage.objects.delete`
+* `storage.objects.get`
+* `storage.objects.list`
+
+**Note:** This can be a custom role, or you can use existing Google Cloud roles `roles/storage.legacyObjectReader` and `roles/storage.legacyBucketWriter`
+
+#### Configure required export project access
+[Add the service account as a principal on the export dataset project resource][7] and assign a role with the following permissions:
+* `bigquery.jobs.create`
+* `bigquery.transfers.get`
+* `bigquery.transfers.update`
+
+**Note:** This can be a custom role, or you can use existing Google Cloud role `roles/bigquery.admin`
+
+#### Configure required export BigQuery dataset access
+[Add the service account as a principal on the export BigQuery dataset resource][8] and assign a role with the following permissions:
+* `bigquery.datasets.get`
+* `bigquery.tables.create`
+* `bigquery.tables.delete`
+* `bigquery.tables.export`
+* `bigquery.tables.get`
+* `bigquery.tables.getData`
+* `bigquery.tables.list`
+* `bigquery.tables.update`
+* `bigquery.tables.updateData`
+
+**Note:** This can be a custom role, or you can use existing Google Cloud role `roles/bigquery.dataEditor`
+
+#### If necessary - configure required cross-project service authorization
+If your integrated Service Account exists in a different Google Cloud Platform project than your billing export dataset you need to [grant cross-project service account authorization][10]:
+
+[Trigger service agent creation][11] using the following values:
```

Comment:
This flows a little weird, are you granting cross project account authorization then Trigger service agent creation ?

Reply:
The overall step is to grant cross-project service account auth.

To do that a user needs to trigger service agent creation which creates a GCP service agent account that has to then be referenced to grant the access